### ISSUE TITLE
REQUEST: *Organism* is missing, can you add it? 

### ISSUE THREAD

**Issue by jespermaag** [jespermaag/gganatogram#17]
# Request for additional organism or organs
Many people have contacted me asking to add their favorite organisms, so I thought I'd create a guiding thread.  

## How organisms were chosen
When I first started working on gganatogram, I realised that it would be impossible for me to draw the anatograms myself, so I set out to use well annotated open source SVG files.   
I remembered seeing some at the Expression Atlas, and decided to use the ones here https://ebi-gene-expression-group.github.io/anatomogram/
I then also found the cell SVG from the Protein Atlas https://www.proteinatlas.org/images_static/cell.svg 

Not all organisms in the Expression Atlas has an anatogram, and most species anatograms are not as well annotated as human and mouse. 
Unfortunately, my drawing and anatomy skills are limited, and gganatogram is a hobby project outside my day job, so I won't be able to manually draw organisms/organs and add them.     

##  How to add a tissue to an existing organism
If your favourite organism is included in gganatogram, but some tissues are missing, how do you add it? 

### Edit SVG in inkscape 
The best way to add an organisms would be to download the SVG from the Expression Atlas (https://github.com/ebi-gene-expression-group/anatomogram/tree/master/src/svg), edit the file and create a new path for your tissue and draw it in inkscape (preferably avoid circles and such). If the tissue is annotated similarly to the tissues in the SVG, it should be easy for me to add the organisms. 

### Add coordinate points 
The easiest for me would be to receive the coordinates (x,y) directly, which I can then read in to R and add to the appropriate organism. 
One way to to this, which I manually did for some tissues for the human anatograms due to my incapability of handling circles in SVG, was to plot the outline of the organisms, and then plot points as outlining the new tissue. I used DescTools's DrawEllipse, which outputs the coordinates of an circle/ellipse.  Otherwise, points that you can connect with a line works,

## How to add a whole new organism
As, I mentioned above, I have no way of drawing a whole new organism from scratch. If you are an expert on a specific organisms, and think adding it to gganatogram will greatly help you to save time. I suggest you either point me to a highly annotated SVG file of said organism, or you commit time/commission to draw it from scratch. 

### I have found a SVG that I want you to add
If you have found an open source SVG that you want me to add, make sure that the anatomy is correct, and that the tissues are names properly. If there's a group of paths, the group name should be the tissue name. 

### I want to draw my favourite organism
Great! I would be happy to add it to gganatogram to help researchers in your field! 

## In summary
How to add a new organism
![image](https://user-images.githubusercontent.com/7098199/47366129-036c9700-d6ab-11e8-8a0e-07488cefbecf.png)
And send the resulting well annotated SVG to me!
